cmd/kube-apiserver/app/options/options.go

```diff
@@ -68,7 +69,7 @@ type ServerRunOptions struct {
 func NewServerRunOptions() *ServerRunOptions {
 	s := ServerRunOptions{
 		GenericServerRunOptions: genericoptions.NewServerRunOptions(),
-		Etcd:                 genericoptions.NewEtcdOptions(kubeoptions.DefaultEtcdPathPrefix, api.Scheme, nil),
+		Etcd:                 genericoptions.NewEtcdOptions(storagebackend.NewDefaultConfig(kubeoptions.DefaultEtcdPathPrefix, api.Scheme, nil)),
```

Comment:
I'd really like keep the arguments to `NewRecommendedOptions` to be: "Where do you want it and what's your scheme".  Even if we construct this under the covers.  I haven't gotten to the point to see if you plumbed it through that way or not, I just thought of it here.